assay/build.gradle

```
@@ -3,3 +3,5 @@ import org.labkey.gradle.util.BuildUtils
 dependencies{
     BuildUtils.addLabKeyDependency(project: project, config: "modules", depProjectPath: BuildUtils.getPlatformModuleProjectPath(project.gradle, "experiment"), depProjectConfig: "published", depExtension: "module")
 }
+
+project.tasks.npmInstall.dependsOn(project.project(":server:modules:platform").tasks.npmInstall)
```

Let's change this to:
```suggestion
project.tasks.npmInstall.dependsOn(project.project(BuildUtils.getPlatformProjectPath(project.gradle).tasks.npmInstall)
```
I don't expect that path to change, but it's nice to have our references centralized.

---

Done. Thanks.